### PR TITLE
Combat Boots

### PR DESCRIPTION
**Pull request by VividPups**
Adds new combat boots to Sec

# Description
how many times have you as a secoff couldn't do your job because of space wind? way too fuckin much if you ask me. So therefor I made these boots. These boots are not only an item Oracle will request but also a Research item under riot control (space riots are real)  Now according to SOP mag boots are a restrictive item for engis meaning that Sec shouldn't be going out of their a way to find them and try and get them (Bad sec off) but, now this with this item Warden and HOS (After research) can regulate who can have them and when it can be handed out..
(Cost of making them can change really just didn't know what to put should probably be more steel that's for sure)

Also, this could help when later down the road we might finally be able to get nukies

# Media
image

# Changelog

:cl:
- add: New Combat Boots!

### CODE REVIEW

**Review by Memeji — APPROVED** [review]

**Review by FoxxoTrystan — APPROVED** [review]
While good is good, Personnaly giving Security Magboots that does not slow down is a no-no for me.

MAGBOOTS SLOWS EVERYONE DOWN EXCEPT THE ADVANCED/EXPERIMENTAL ONES witch should be UNIQUE.

But regarding SOP that apprently "not everyone can use magboots" for me that bullshit too, magboots should be buildable in science at peoples request, ect...

I will put this under maintainer review.

**Comment by VividPups** [comment]
> While good is good, Personnaly giving Security Magboots that does not slow down is a no-no for me.
> 
> MAGBOOTS SLOWS EVERYONE DOWN EXCEPT THE ADVANCED/EXPERIMENTAL ONES witch should be UNIQUE.
> 
> But regarding SOP that apprently "not everyone can use magboots" for me that bullshit too, magboots should be buildable in science at peoples request, ect...
> 
> I will put this under maintainer review.

The reason why I put it as no slow 

> While good is good, Personnaly giving Security Magboots that does not slow down is a no-no for me.
> 
> MAGBOOTS SLOWS EVERYONE DOWN EXCEPT THE ADVANCED/EXPERIMENTAL ONES witch should be UNIQUE.
> 
> But regarding SOP that apprently "not everyone can use magboots" for me that bullshit too, magboots should be buildable in science at peoples request, ect...
> 
> I will put this under maintainer review.

The reason why I put it at no slow down is because  amount of mobs that way to fast to have it like the regular the only real threats are the dragon, ninja and ratking all which have significantly faster and can kill their enemy. Also, I have not once have seen Advance or experimental magboots. (Unless you are talking about the single boots that CE has)

**Comment by FoxxoTrystan** [comment]
> > While good is good, Personnaly giving Security Magboots that does not slow down is a no-no for me.
> > MAGBOOTS SLOWS EVERYONE DOWN EXCEPT THE ADVANCED/EXPERIMENTAL ONES witch should be UNIQUE.
> > But regarding SOP that apprently "not everyone can use magboots" for me that bullshit too, magboots should be buildable in science at peoples request, ect...
> > I will put this under maintainer review.
> 
> The reason why I put it as no slow
> 
> > While good is good, Personnaly giving Security Magboots that does not slow down is a no-no for me.
> > MAGBOOTS SLOWS EVERYONE DOWN EXCEPT THE ADVANCED/EXPERIMENTAL ONES witch should be UNIQUE.
> > But regarding SOP that apprently "not everyone can use magboots" for me that bullshit too, magboots should be buildable in science at peoples request, ect...
> > I will put this under maintainer review.
> 
> The reason why I put it at no slow down is because amount of mobs that way to fast to have it like the regular the only real threats are the dragon, ninja and ratking all which have significantly faster and can kill their enemy. Also, I have not once have seen Advance or experimental magboots. (Unless you are talking about the single boots that CE has)

Yes CE boots are normally unique, we are reviewing this and still go back to you.

File: Resources/Prototypes/Floof/Entities/Clothing/Shoes/Combatmags.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/Shoes/Combatmags.yml:2]:
```suggestion
  parent: ClothingShoesBootsMag
```

File: Resources/Prototypes/Floof/Entities/Clothing/Shoes/Combatmags.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/Shoes/Combatmags.yml:5]:
```suggestion
  description: Made by SESWC. These State-of-the-art magnetic boots with built in armor for extra protection
```
Edit description as needed.

File: Resources/Prototypes/Research/arsenal.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Research/arsenal.yml:116]:
```suggestion
```

File: Resources/Prototypes/Floof/Recipes/Lathes/security.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Recipes/Lathes/security.yml:24]:
```suggestion
  completetime: 10
  materials:
    Steel: 1000
    Plastic: 500
```

Same cost as Science/Normal Magboots, feel free to add extra due to the small armor.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
After a tiny talk here what i suggest, first advanced magboots is a nono but giving modified magboots for combat is fine to the security lathe, just make sure cost is not below normal magboots and make the research instead under "MagnetsTech", you will need to move ClothingShoesBootsMagCom under there.

Also i suggest that you could give them a custom/different sprite: here one from Frontier that could work: https://github.com/new-frontiers-14/frontier-station-14/tree/master/Resources/Textures/_NF/Clothing/Shoes/Boots/magboots-combat.rsi

Suggestion to assit you are below, if need assistance do tell.

File: Resources/Prototypes/Research/arsenal.yml
Comment (FoxxoTrystan) [reply Resources/Prototypes/Research/arsenal.yml:116]: Move this to MagnetsTech Instead.

**Comment by VividPups** [comment]
> After a tiny talk here what i suggest, first advanced magboots is a nono but giving modified magboots for combat is fine to the security lathe, just make sure cost is not below normal magboots and make the research instead under "MagnetsTech", you will need to move ClothingShoesBootsMagCom under there.
> 
> Also i suggest that you could give them a custom/different sprite: here one from Frontier that could work: https://github.com/new-frontiers-14/frontier-station-14/tree/master/Resources/Textures/_NF/Clothing/Shoes/Boots/magboots-combat.rsi
> 
> Suggestion to assit you are below, if need assistance do tell.

the problem is again what is going to happen is that current antags are going to be much faster than the Secoff trying to defend the station which will just cost more deaths

**Comment by VividPups** [comment]
> Current antags has the same problem as security regarding gravity, Magboots are toggable. wana walk normal speed? toggle it off.
> 
> No Gravity? Toggle it on.

and then you trip and die, or you toggle the antag gets in your range and kills you that's the problem.  it happen last night with a loneop

**Comment by FoxxoTrystan** [comment]
> > Current antags has the same problem as security regarding gravity, Magboots are toggable. wana walk normal speed? toggle it off.
> > No Gravity? Toggle it on.
> 
> and then you trip and die, or you toggle the antag gets in your range and kills you that's the problem. it happen last night with a loneop

Then be carefull, even loneop/Syndicate magboots are not advanced.

**Comment by VividPups** [comment]
all Request has been made

File: Resources/Prototypes/Floof/Entities/Clothing/Shoes/Combatmags.yml
Comment (FoxxoTrystan) [reply Resources/Prototypes/Floof/Entities/Clothing/Shoes/Combatmags.yml:9]: This need to be removed.
Pretty sure that is also not compatible with magboots.

Comment (VividPups) [reply Resources/Prototypes/Floof/Entities/Clothing/Shoes/Combatmags.yml:9]: those are when you aware it and I did remove it

**Review by FoxxoTrystan — APPROVED** [review]

**Review by Memeji — APPROVED** [review]

**Review by Fansana — CHANGES_REQUESTED** [review]
The shoes will need their own sprite. Working on it, do not merge.

**Review by Fansana — APPROVED** [review]

**Pull request merged**
